primap2/tests/data/BURDI_conversion.csv

```diff
@@ -17,16 +17,19 @@ BURDI,IPCC2006_PRIMAP,comment
 2.C,2.C
 2.F,2.F
 2.G + 2.D, 2.H
+2.G, 2.H.3
 3,2.D
 4,M.AG
 4.A,3.A.1
 4.B,3.A.2
 4.C,3.C.7
+4.D, 3.C.45.AG
```

Comment:
This will not work unless `3.C.45.AG` is added to the IPCC2006_PRIMAP conversion in climate categories.

Reply:
Updated to `M.3.C.45.AG`, here and in https://github.com/primap-community/climate_categories/pull/169